### PR TITLE
Change documentation website to new layout

### PR DESCRIPTION
**Pull request by flexdinesh**
Changes include
- Separate layout for the website and docs
- Docs now have a fixed header, a left nav and a scrollable content container
- Border to separate header, left nav and the content

### CODE REVIEW

File: docs/components/Page.tsx [hunk 1]
```diff
@@ -81,51 +81,66 @@ export function DocsPage({
...
-              css={{ justifyContent: 'space-between', alignItems: 'baseline' }}
+            <main
+              id="skip-link-content"
+              tabIndex={0}
+              ref={contentRef}
+              className={noProse ? '' : 'prose'}
+              css={{
+                paddingTop: '2rem',
+                // to prevent default grid item min-width: auto behaviour
+                // it makes pre tags looooooong
```
Comment (dcousens) [inline docs/components/Page.tsx:121]: is this comment in reference to `paddingTop`?

Comment (flexdinesh) [reply docs/components/Page.tsx:121]: Yeah nah. This comment should go away. Added this comment before I figured out that minmax(0, 1fr) on grid container is enough to contain grid items to viewport width. I previously used minWidth: 0 on grid items but missed removing the comment when I made the change. Will remove it now.

Comment (flexdinesh) [reply docs/components/Page.tsx]: done

**Review by dcousens — APPROVED** [review]

File: docs/components/Footer.tsx [hunk 2]
```diff
@@ -264,3 +264,54 @@ export function Footer() {
...
+            // textAlign: 'center',
+          },
+        })}
+      >
+        <Type look="body14" as="p" css={{ justifySelf: 'start' }}>
+          Made in Australia <Emoji symbol="🇦🇺" alt="Australia" /> by{' '}
+          <a href="https://www.thinkmill.com.au" target="_blank" rel="noopener noreferrer">
+            Thinkmill
+          </a>
+          . Contributed to around the world <Emoji symbol="🌏" alt="Globe" />
```
Comment (dcousens) [inline docs/components/Footer.tsx:303]:
```suggestion
          , with contributions from around the world <Emoji symbol="🌏" alt="Globe" />
```

**Pull request merged**